include/view-impl.h

```diff
@@ -0,0 +1,14 @@
+/* SPDX-License-Identifier: GPL-2.0-only */
+#ifndef __LABWC_VIEW_IMPL_INTERNAL_H
```

Comment:
Should we use this opportunity to rename this to `view-common.h` or `view-impl-common.h` ?

Reply:
`git mv view-impl.c view-impl-common.c` done.